libraries/botbuilder-ai/tests/qnaMaker.test.js

```diff
@@ -800,3 +901,30 @@ class overrideTwoEventsWithOverrideLogger extends QnAMaker
             });
     }
 }
+
+class TrainApiHelper
+{
+    constructor() {
```

Comment:
Lambda that returns an object? Do we need to have the class since the constructor doesn't take anything.

Reply:
answer: no, no class needed, indeed!

Just moved variables to the `describe('Active Learning'...)` level, so it's accessible to all test added here